src/Compilers/Core/Portable/Diagnostic/DiagnosticDescriptor.cs

```diff
@@ -227,6 +227,8 @@ private static ReportDiagnostic MapSeverityToReport(DiagnosticSeverity severity)
                     return ReportDiagnostic.Warn;
                 case DiagnosticSeverity.Error:
                     return ReportDiagnostic.Error;
+                case DiagnosticSeverity.Suggestion:
```

Comment:
hmmm.. difference seems quite hard to get. isn't unnecessary a suggestion?

among, error, warning, suggestion, info, hidden. I can understand error and warning. what are difference between suggestion, info, hidden?

for me all seems same thing with desire to have different presentation.